oc-chef-pedant/lib/pedant/rspec/search_util.rb

```diff
@@ -950,7 +950,7 @@ def do_search(type, name, should_find=true)
         should_not_find_any(temporary_data_bag_name)
 
         # Now, send everything to be re-indexed
-        `#{executable} reindex #{reindex_args.join(" ")} #{Pedant::Config.reindex_endpoint}`
+        puts `#{executable} reindex #{reindex_args.join(" ")} #{Pedant::Config.reindex_endpoint}`
```

Comment:
These tests fail often enough in the normal case that the addional output can only help.